virttest/vt_resmgr/resources/storage/dir/dir_volume.py

```
@@ -0,0 +1,52 @@
+        r, o = node.proxy.resource.create_resource_backing(
+            self.get_backing_config(node.name)
+        )
+        if r != 0:
+            raise Exception(o["out"])
+
+        self._update_binding(node, o["out"]["backing"])
+        self._update_uri(node.name, o["out"]["spec"]["uri"])
+
+    def unbind_backings(self, nodes):
+        if not self.binding_backings:
+            raise RuntimeError(f"Cannot unbind an unbound dir volume {self.name}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)